docs/ipfs.md

```diff
@@ -159,14 +167,19 @@ When you specify `--ipfs` option to `nerdctl build`, it automatically starts the
 By default, nerdctl exposes the registry at `localhost:5050`.
 You can change the address and can manually restart the registry using `nerdctl ipfs registry up` and `nerdctl ipfs registry down`.
 
+To make these commands work correctly, you need to install `nerdctl ipfs registry serve` command as a systemd service named `nerdctl-ipfs-registry-serve`.
```

Comment:
Maybe we can have a sample `systemd service` file that can be used directly by users?